src/main/java/nextstep/ladder/domain/PointGenerator.java

```diff
@@ -1,47 +1,48 @@
 package nextstep.ladder.domain;
 
+import nextstep.ladder.util.RandomUtil;
+
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
-import java.util.Random;
+import java.util.stream.Collectors;
 
 public class PointGenerator {
 
     private static final int START_IDX = 1;
 
-    private static List<Point> points;
+    private List<Point> points;
+
+    protected PointGenerator() {}
```

Comment:
```suggestion
    protected PointGenerator() {
        this(new ArrayList<>());
    }
```
모든 생성자는 적어도 다른 생성자 하나를 호출하도록 하는게 유지보수상 좋습니다. 😀  이렇게 바꾸면 25번 라인드 필요없어지겠네요.